controllers/product.js

```diff
@@ -11,4 +11,15 @@ const productController = async (req, res) => {
   }
 };
 
-module.exports = { productController };
+const productDetailController = async (req, res) => {
```

Comment:
```suggestion
const getProductDetail = async (req, res) => {
```
이미 파일 자체가 컨트롤러 파일이라서, 컨트롤러 이름은 없어도 될 것 같습니다 :)

Reply:
네~ 참고해서 수정하겠습니다 감사합니다~!